trestle/core/commands/author/jinja.py

```diff
@@ -117,13 +115,16 @@ def jinja_ify(
         try:
             if lut is None:
                 lut = {}
+            template_folder = pathlib.Path.cwd()
+            if len(r_input_file.parts) > 1:
+                template_folder = '/'.join(r_input_file.parts[0:-1]).replace('//', '/')
```

Comment:
Looking at this line again. You might need to explain to me why we need this. Pathlib seems to get rid of '//' for me - unless that is a py3.9 feature.